CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [vUNRELEASED]
+
+- rkt/export: Add new subcommand `export` ([#2343](https://github.com/coreos/rkt/pull/2343)). Export a stopped, single-pod container, along with its state at the time of exit, to an ACI.
```

Comment:
Single pod container: do you mean single app pod?

What is the state that is included?